assets/components/organisms/footer/footer-legal.twig

```diff
@@ -15,7 +15,7 @@
 <div class="footer-legal">
   <div class="footer-legal-links">
     <a href="https://epfl-idevelop.github.io/elements/#/doc/accessibility.html">Accessibility</a>
-    <a href="https://mediacom.epfl.ch/disclaimer">Legals</a>
+    <a href="https://www.epfl.ch/about/overview/overview/regulations-and-guidelines/">Legals</a>
```

Comment:
Selon https://www.epfl.ch/campus/services/en/it-services/web-services/legal-recommendations-for-publishing-on-the-epfl-web/use-of-google-analytics/ -> Ajouter un lien vers les mentions légales (obligatoire), le texte devrait être : "Legal notice"